web-registry/.env.example

```diff
@@ -8,6 +8,6 @@ REACT_APP_SANITY_DATASET=
 REACT_APP_SANITY_PROJECT_ID=
 REACT_APP_SANITY_TAG=default
 REACT_APP_LEDGER_CHAIN_ID=
-REACT_APP_LEDGER_REST_ENDPOINT=
```

Comment:
using the registry-server proxies so no need to have a direct reference to this anymore from the front-end